horace_core/sqw/@axes_block/private/calc_bin_nodes_.m

```diff
@@ -81,17 +130,40 @@
     nodes = [Xn(:)';Yn(:)';Zn(:)';En(:)'];
 end
 
-function char_size = parse_inputs(ninputs,varargin)
+function char_size = parse_inputs(noptions,ninputs,varargin)
```

Comment:
Again, why use `varargin`, if there's only one valid option (`cube`)?

Reply:
because I've tried different options and thinking about expansion and adding `-` controlled options